test/unit/system/prepare_test.py

```diff
@@ -299,7 +299,7 @@ def test_install_bootstrap(
         mock_tar.assert_called_once_with(
             '{0}/bootstrap.tgz'.format(self.description_dir)
         )
-        tar.extract.assert_called_once_with('root_dir')
+        tar.extract.assert_called_once_with('root_dir/foo')
```

Comment:
I think it would be good if the test that extracts an archive without a target_dir continues to stay. That way we can ensure the change does not break the "default" behavior. It would be imho good to just have one additional test to explicitly test the case with target_dir. What do you think ?